cli_tools/gce_vm_image_import/importer/inflater.go

```diff
@@ -150,3 +152,46 @@ func (inflater *daisyInflater) cancel(reason string) bool {
 	inflater.wf.CancelWithReason(reason)
 	return true
 }
+
+func createDaisyVarsForFile(args ImportArguments,
+	fileInspector vdisk.VirtualDiskFileInspector, diskName string) map[string]string {
+	vars := map[string]string{
+		"source_disk_file": args.Source.Path(),
+		"import_network":   args.Network,
+		"import_subnet":    args.Subnet,
+		"disk_name":        diskName,
+	}
+
+	// To reduce the runtime permissions used on the inflation worker, we pre-allocate
+	// disks sufficient to hold the disk file and the inflated disk. If inspection fails,
+	// then the default values in the daisy workflow will be used. The scratch disk gets
+	// a padding factor to account for filesystem overhead.
+	deadline, cancelFunc := context.WithDeadline(context.Background(), time.Now().Add(inspectionTimeout))
+	defer cancelFunc()
+	metadata, err := fileInspector.Inspect(deadline, args.Source.Path())
+	if err == nil {
+		vars["inflated_disk_size_gb"] = fmt.Sprintf("%d", calculateInflatedSize(metadata))
+		vars["scratch_disk_size_gb"] = fmt.Sprintf("%d", calculateScratchDiskSize(metadata))
+	}
+	return vars
+}
+
+// Allocate extra room for filesystem overhead, and
+// ensure a minimum of 10GB (the minimum size of a GCP disk).
+func calculateScratchDiskSize(metadata vdisk.VirtualDiskFileMetadata) int64 {
+	// This uses the historic padding calculation from import_image.sh: add ten percent,
+	// and round up.
+	padded := int64(float64(metadata.PhysicalSizeGB)*1.1) + 1
+	if padded < 10 {
+		return 10
+	}
+	return padded
```

Comment:
Shouldn't the +1 be within int64 conversion since the PhysicalSizeGB*1.1 could be a round number?

Also, any reason not to use math.Min and math.Ceil combo? 

Something like:
return int64(math.Min(10, math.Ceil(float64(metadata.PhysicalSizeGB)*1.1))

Reply:
I originally did the more accurate approach that you described, but found that the inflation worker was resizing the disk when `PhysicalSizeGB*1.1` was a whole number. To avoid that, I switched to emulate the calculation from the worker:


https://github.com/GoogleCloudPlatform/compute-image-tools/blob/master/daisy_workflows/image_import/import_image.sh#L93

Reply:
You can still use math.min to achieve this logic if you move the +1 within float64. It's really minor but less lines if reasonably readable is preferred. 

Also, 10 should be a constant as it's reused. I thought I made a comment about it but don't see it now.

Reply:
Regarding math.min, I'd prefer to use the same/similar code in the two implementations for clarity to the reader if that's alright with you. (Especially since unit testing that bash code is not an option right now.)

For the `10` constant, perhaps try reloading the page? You can also search for `defaultInflationDiskSizeGB` here:

https://github.com/GoogleCloudPlatform/compute-image-tools/pull/1283/commits/2bcf36860c6d1a027db3c39fd65b5cf29e467eb0